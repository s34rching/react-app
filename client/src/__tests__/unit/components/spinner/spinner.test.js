import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../../../components/spinner/spinner.component';

describe('Components: Spinner', () => {
  it('should render without errors', () => {
    const wrapper = shallow(<Spinner />);

    expect(wrapper).toMatchSnapshot();
  });
});
