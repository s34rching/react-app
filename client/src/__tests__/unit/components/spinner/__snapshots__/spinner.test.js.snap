// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: Spinner should render without errors 1`] = `
<SpinnerOverlay>
  <SpinnerContainer />
</SpinnerOverlay>
`;
